specification/subscription/resource-manager/Microsoft.Subscription/preview/2021-01-01-privatepreview/subscriptions.json

```diff
@@ -0,0 +1,1469 @@
+          "description": "The friendly name of the subscription.",
+          "type": "string"
+        },
+        "skuId": {
+          "description": "The SKU ID of the Azure plan. Azure plan determines the pricing and service-level agreement of the subscription.  Use 001 for Microsoft Azure Plan and 002 for Microsoft Azure Plan for DevTest.",
+          "type": "string"
+        },
+        "resellerId": {
+          "description": "Reseller ID, basically MPN Id.",
+          "type": "string"
+        }
+      }
+    },
+    "Workload": {
+      "description": "The workload type of the subscription. It can be either Production or DevTest.",
+      "type": "string",
+      "x-ms-enum": {
+        "name": "workload",
+        "modelAsString": true
+      },
+      "enum": [
+        "Production",
+        "DevTest"
+      ]
+    },
+    "BillingScope": {
+      "description": "Billing scope of the subscription.\nFor CustomerLed and FieldLed - /billingAccounts/{billingAccountName}/billingProfiles/{billingProfileName}/invoiceSections/{invoiceSectionName}\nFor PartnerLed - /billingAccounts/{billingAccountName}/customers/{customerName}\nFor Legacy EA - /billingAccounts/{billingAccountName}/enrollmentAccounts/{enrollmentAccountName}",
+      "type": "string"
+    },
+    "AcceptOwnershipState": {
+      "readOnly": true,
+      "type": "string",
+      "description": "The accept ownership state of the resource.",
+      "enum": [
+        "Pending",
+        "Completed",
+        "Expired"
+      ],
+      "x-ms-enum": {
+        "name": "AcceptOwnership",
+        "modelAsString": true
+      }
+    },
+    "ServiceTenantResponse": {
+      "description": "Billing account service tenant.",
+      "properties": {
+        "tenantId": {
+          "description": "Service tenant id.",
+          "type": "string"
+        },
+        "tenantName": {
+          "description": "Service tenant name.",
+          "type": "string"
+        }
+      }
+    }
+  },
+  "parameters": {
+    "apiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Version of the API to be used with the client request. Current version is 2021-01-01-privatepreview"
+    },
+    "subscriptionIdParameter": {
+      "name": "subscriptionId",
+      "x-ms-parameter-location": "method",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Subscription Id."
+    },
+    "subscriptionNameParameter": {
+      "name": "body",
+      "x-ms-parameter-location": "method",
+      "in": "body",
+      "required": true,
+      "description": "Subscription Name",
+      "schema": {
+        "$ref": "#/definitions/SubscriptionName"
+      }
+    },
+    "aliasNameParameter": {
+      "name": "aliasName",
+      "x-ms-parameter-location": "method",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Alias Name"
```

Comment:
Can we change description to this - "“AliasName – Name for this subscription creation request also known as alias. Note that this is not the same as subscription name and this doesn’t have any other lifecycle need beyond the request for subscription creation.”

Reply:
Can we change description in older versions too

Reply:
Done.